docs/manual/external-tools.tex

```diff
@@ -537,6 +538,34 @@ \section{Javac compiler\label{javac-installation}}
 % \end{itemize}
 
 
+\section{Lombok\label{lombok}}
+
+Project Lombok (\url{https://projectlombok.org/}) is ``a java library that
```

Comment:
Nit: capitalize "Java"

Reply:
This is a quote from the Lombok website.

Reply:
(The useless part is "spicing up your java".)